pages/common/assets/sass/style.scss

```diff
@@ -54,20 +54,26 @@
   }
 }
 
+#activity .toggle-switch {
```

Comment:
I'd generally try to avoid using id selectors in CSS ever as they're super specific and so become really difficult to extend, which leads to slightly hateful code in future.

Reply:
I need to only select the toggle switch on this page, rather than all toggle-switches - should I just make the class something like `activity-toggle-switch` ?

Reply:
Just target the container with a classname instead of the id. You might need to replace the wrapping `<Fragment>` below with a `<div className="activity-log">` to do this.